my-app/src/components/membersCardsSection/MembersCardsSection.jsx

```diff
@@ -0,0 +1,30 @@
+import { MemberCard } from "../memberCard/MemberCard";
+import Search from "../search/Search";
+import "./MembersCardsSection.css";
+
+export const MembersCardsSection = ({ showSearch }) => {
+  const members = [
+    { name: "Carolien Bloeme", selected: false },
+    { name: "Sun Jun", selected: true },
+    { name: "Song Bao", selected: false },
+    { name: "Olivia Arribas", selected: true },
+    { name: "Bonginkosi Mdladlana", selected: false },
+    { name: "Arina Belomestnykh", selected: true },
+    { name: "Jacqueline Likoki", selected: true },
+  ];
+
+  return (
+    <>
+      {showSearch && <Search />} 
+      <div className="member-list">
+        {members.map((member, index) => (
```

Comment:
Just to make this more realistic, we do not usually depend on the index as a key for the mapped items in React. You can add an id key to the members  list and give it random values and use it here instead.

Reply:
Done